gcm_filters/filter.py

```diff
@@ -421,14 +423,14 @@ def plot_shape(self, ax=None):
             linewidth=4,
         )
         ax.axvline(
-            2 * np.pi / self.filter_scale,
+            2 * np.pi / self.filter_scale_iterated,
```

Comment:
On the lines above, the target that is plotted is `F(x) ** self.n_iterations` and the approximation that is plotted is `np.polynomial.chebyshev.chebval(x, self.filter_spec.p) ** self.n_iterations`. For these plots I think we want the vertical line to be at `filter_scale` not `filter_scale_iterated` right?